### PR TITLE
Adding try-with-resources block around PredictionTraverser

### PR DESCRIPTION
**Pull request by vamossagar12**
Fixes https://github.com/apache/incubator-hugegraph/issues/1818

### CODE REVIEW

**Comment by vamossagar12** [comment]
@javeme , another small PR. Plz review. Thanks!

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java [hunk 1]
```diff
@@ -80,9 +80,10 @@ public String get(@Context GraphManager manager,
         Directions dir = Directions.convert(EdgeAPI.parseDirection(direction));
 
         HugeGraph g = graph(manager, graph);
-        PredictionTraverser traverser = new PredictionTraverser(g);
-        double score = traverser.adamicAdar(sourceId, targetId, dir,
-                                            edgeLabel, maxDegree, limit);
-        return JsonUtil.toJson(ImmutableMap.of("adamic_adar", score));
+        try (PredictionTraverser traverser = new PredictionTraverser(g)) {
+            double score = traverser.adamicAdar(sourceId, targetId, dir,
+                edgeLabel, maxDegree, limit);
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java:85]: prefer to keep the origin alignment

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/ResourceAllocationAPI.java [hunk 2]
```diff
@@ -80,10 +80,11 @@ public String create(@Context GraphManager manager,
...
 
         HugeGraph g = graph(manager, graph);
-        PredictionTraverser traverser = new PredictionTraverser(g);
-        double score = traverser.resourceAllocation(sourceId, targetId, dir,
-                                                    edgeLabel, maxDegree,
-                                                    limit);
-        return JsonUtil.toJson(ImmutableMap.of("resource_allocation", score));
+        try(PredictionTraverser traverser = new PredictionTraverser(g)) {
+            double score = traverser.resourceAllocation(sourceId, targetId, dir,
+                edgeLabel, maxDegree,
```
Comment (javeme) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/ResourceAllocationAPI.java:85]: prefer to keep the origin alignment

**Review by javeme — COMMENTED** [review]
Thanks for your contribution~

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java (hunk 1 above)
Comment (imbajin) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java:85]: if one line < 100 chars, use one line is fine (and we have a code-style setting file in root for IDEA, use it to modify code is much more easy)

Comment (vamossagar12) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java]: fixed the alignment. Thanks for the tip on code-style setting file.

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/ResourceAllocationAPI.java (hunk 2 above)
Comment (vamossagar12) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/ResourceAllocationAPI.java]: fixed.

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Comment by javeme** [comment]
I've updated the branch on the github page that caused the author is changed, which doesn't seem to be expected behavior.
rebase-update

@vamossagar12 can you please rebase on the latest master by `git rebase -i master` and `git push -f` again?

**Comment by vamossagar12** [comment]
Thanks @javeme , @imbajin !

**Pull request merged**